docs/getting-started.md

```diff
@@ -0,0 +1,171 @@
+# Getting Started
+
+- [Install](#install)
+- [Create an API](#create-an-api)
+
+Tyk Operator extends Kubernetes API with Custom Resources. API Definitions, Security Policies, Authentication, 
+Authorization, Rate Limits, and other Tyk features can be managed just like other native Kubernetes objects, leveraging 
+all the features of Kubernetes like kubectl, security, API services, RBAC. You can use Tyk Customer Resources to manage 
```

Comment:
Small typo here. It should be Tyk Custom Resources.

Reply:
ups. i fixed it, thanks 👍